pkgs/by-name/ja/jawiki-all-titles-in-ns0/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+}:
+
+let
+date = "20240520";
+jawiki = fetchurl {
+  url ="https://dumps.wikimedia.org/jawiki/${date}/jawiki-${date}-all-titles-in-ns0.gz";
+  hash= "sha256-HMQutYuKiI7fXmbivS0mIgdHsQu6t6WdVFVrnyw8JIw=";
+};
+in stdenvNoCC.mkDerivation {
+  pname = "jawiki-all-titles-in-ns0";
+  version = date;
```

Comment:
Versions specified by date should be formatted like `0-unstable-YYYY-MM-DD`

Reply:
I'll fix this to "0-unstable-${date}" for now